### PR TITLE
factored common stuff in percentageValue gsp file

### PR DESCRIPTION
**Pull request by fterrier**
could not get rid of the 2 templates because of the link being different, but factored common stuff in another template

### CODE REVIEW

**Comment by fterrier** [comment]
i guess the link could be passed as a parameter to the template ?

File: grails-app/views/dashboard/_reportProgramTable.gsp [hunk 1]
```diff
@@ -10,48 +10,18 @@
...
-					<!-- comparison value -->
-					<div id="compare-dashboard-entity-${entity.id}" 
-					class="js_bar_horizontal tooltip horizontal-bar-avg" 							
-						data-percentage="45" 
-						style="width:45%;" 
-						original-title="45%"></div>
+						<g:i18n field="${entity.names}" />
+					</g:else>
 				</td>
+				<g:render template="percentageValue" model="[percentageValue: dashboard.getPercentage(currentLocation, entity), id: entity.id]"/>
```
Comment (susanm74) [inline grails-app/views/dashboard/_reportProgramTable.gsp:24]: can the 'id' param be renamed to something a bit more descriptive?

File: grails-app/views/dashboard/_reportLocationTable.gsp [hunk 2]
```diff
@@ -10,54 +10,20 @@
...
-						<div class="js_bar_horizontal tooltip horizontal-bar expand-bar"
-							data-entity="${currentProgram.id}"
-							data-percentage="${percentageValue}"
-							style="width:100%;"
-							original-title="${percentageValue}%"></div>
-					</g:else> 					
+						<g:i18n field="${location.names}" />
+					</g:else>
 				</td>
+				<g:render template="percentageValue" model="[percentageValue: dashboard.getPercentage(location, dashboardEntity), id: location.id]"/>
```
Comment (susanm74) [inline grails-app/views/dashboard/_reportLocationTable.gsp:24]: can the 'id' param be renamed to something a bit more descriptive?

**Comment by fterrier** [comment]
sure, pull the branch and do whatever you want with it :-)

**Pull request closed without merging**